chickadee/processes/wps_CI.py

```diff
@@ -0,0 +1,144 @@
+from pywps import Process, ComplexInput, ComplexOutput, LiteralInput, FORMATS
+from pywps.app.Common import Metadata
+from pywps.app.exceptions import ProcessError
+
+from wps_tools.utils import log_handler
+from wps_tools.io import log_level
+from chickadee.utils import logger, get_ClimDown
+
+
+class CI(Process):
+    def __init__(self):
+        self.status_percentage_steps = {
+            "start": 0,
+            "process": 10,
+            "build_output": 95,
+            "complete": 100,
+        }
+        inputs = [
+            log_level,
+            ComplexInput(
+                "gcm_file",
+                "GCM NetCDF file",
+                abstract="Filename of GCM simulations",
+                min_occurs=1,
+                max_occurs=1,
+                supported_formats=[FORMATS.NETCDF],
```

Comment:
I imagine we would also want to accept `gcms` from `thredds`. That _should_ be an option here unless there is some technical holdup I am not aware of.

Reply:
Forgot about that, nice catch!